perfkitbenchmarker/flag_util.py

```diff
@@ -190,3 +191,149 @@ def DEFINE_integerlist(name, default, help, on_nonincreasing=None,
   serializer = IntegerListSerializer()
 
   flags.DEFINE(parser, name, default, help, flag_values, serializer, **args)
+
+
+SI_SUFFIXES = ['', 'K', 'M', 'G', 'T', 'P']
```

Comment:
Considering there's apparent support for PB, you should add some test cases and verify it actually works. I imagine there will be precision issues, since Python only guarantees at least 32 bits for an int.